lib/datadog/tracing/distributed/b3.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+# typed: true
+
+require_relative 'ext'
+require_relative 'helpers'
+require_relative '../trace_digest'
+
+module Datadog
+  module Tracing
+    module Distributed
+      # B3-style trace propagation.
+      # @see https://github.com/openzipkin/b3-propagation#multiple-headers
+      class B3
+        def initialize(
+          fetcher:,
+          trace_id_key: Ext::B3_HEADER_TRACE_ID,
+          span_id_key: Ext::B3_HEADER_SPAN_ID,
+          sampled: Ext::B3_HEADER_SAMPLED
```

Comment:
Minor: Suggest calling it `sampled_key`, to align with the two others :)

Reply:
I missed this one, good catch!